src/ocean/util/encode/Base64.d

```diff
@@ -507,6 +512,37 @@ unittest
         test!("==")(result, payload);
         result = decode("SGVsbG8sIGhvdyBhcmUgeW91IHRvZGF5Pw==");
         test!("==")(result, payload);
+        result = decode("SGVsbG8sIGhvdyBhcmUgeW91IHRvZGF5Pw=");
+        test!("==")(result, payload);
+        result = decode("SGVsbG8sIGhvdyBhcmUgeW91IHRvZGF5Pw");
+        test!("==")(result, payload);
+    }
+
+    // single padding test
+    {
+        // try the string with the single `=` padding, and without it
+        auto result = cast(Const!(ubyte)[])"any carnal pleasure.";
+        test!("==")(result, decode("YW55IGNhcm5hbCBwbGVhc3VyZS4="));
+        test!("==")(result, decode("YW55IGNhcm5hbCBwbGVhc3VyZS4"));
+    }
+
+    // short decode test
+    {
+        ubyte[1024] decoded;
```

Comment:
Ah, its a copy/paste error. I will remove. `decode` has overflow which will allocate its own buffer, I think that's fine for this unittest.